File: PHP PowerPoint Examples/ImagesApiDemo/getPptImagesList.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\ImagesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ImagesApi($configuration);

$name = "GetPptImagesList.pptx";
$password = null;
$folder = "input";
$storage = null;
$result = $apiInstance->getPptImagesList($name, $password, $folder, $storage);
print_r($result);

File: Python PowerPoint Examples/ImagesApiDemo/getPptImagesList.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.images_api.ImagesApi(configuration)

name = "GetPptImagesList.pptx"
password = ""
storage = ""
folder = "input"

result = api.get_ppt_images_list(name=name, password=password, storage=storage,
                                 folder=folder)
